scheduler/src/cook/rest/api.clj

```diff
@@ -730,7 +738,14 @@
                                                                 (str/lower-case operator))
                                   :constraint/pattern pattern}]))
                             constraints)
-        container (if (nil? container) [] (build-container user db-id container))
+        pool-name (or (:pool/name pool) (config/default-pool))
+        default-container-for-pool (get-in config/config [:settings :pools :default-container-for-pool])
+        container (if (nil? container)
+                    (let [guessed-container (match-default-container-for-pool default-container-for-pool pool-name)]
```

Comment:
Maybe `default-container` instead of `guessed-container`?

Reply:
Changed.